src/TMap.cpp

```diff
@@ -2584,3 +2585,11 @@ void TMap::update()
     }
 }
 
+QString TMapLabel::base64EncodePixmap()
+{
+    QBuffer buffer;
+    buffer.open(QIODevice::WriteOnly);
+    pix.save(&buffer, "PNG");
+    auto const encoded = buffer.data().toBase64();
```

Comment:
I.e. 
```suggestion
    return buffer.data().toBase64();
```
and loose the following line...

Reply:
Corrected, works like charm :)